src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -47,6 +47,13 @@ internal class WinHttpHandler : HttpMessageHandler
     public class WinHttpHandler : HttpMessageHandler
 #endif
     {
+#if NET46
+        internal static Version HttpVersion20 => new Version(2,0);
+        internal static Version HttpVersionUnknown => new Version(0,0);
```

Comment:
Should these instances be cached in private static readonly fields? Otherwise, a new instance will be allocated each time the property is accessed.